### PR TITLE
[DJFS-153] Article Publishing Endpoints

### PR DESCRIPTION
**Pull request by Israel-Force**
## Task: `Article Publishing Endpoints`
- `added a .txt file containing the link to my api documentation`

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done @Israel-Force 

Aside from `GET` request, it is not advisable to pass your `id` on the URI as seen here `localhost:8080/api/article/5`. pass the `id` is the `Body` as a post data. Also, you need to push your code along with your PR.

Remove the `PULL_REQUEST_TEMPLATE.md` that you pushed along with your PR.

Lastly, kindly give a thumbs up if you find the article helpful. Thanks

**Comment by Israel-Force** [comment]
> Well done @Israel-Force
> 
> Aside from `GET` request, it is not advisable to pass your `id` on the URI as seen here `localhost:8080/api/article/5`. pass the `id` is the `Body` as a post data. Also, you need to push your code along with your PR.
> 
> Remove the `PULL_REQUEST_TEMPLATE.md` that you pushed along with your PR.
> 
> Lastly, kindly give a thumbs up if you find the article helpful. Thanks

Thanks. But I am not sure I got what you are trying to say. You said aside from get request, the route you alighted is from a get request. Can you please try and throw more light.

**Review by adaugochi — CHANGES_REQUESTED** [review]
You are removing these files. Kindly revert them back

image

**Comment by Israel-Force** [comment]
I don't even know how they got there not to talk of removing them @adaugochi

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly visit your PR https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/pull/1284/files and see the changes you made. So you to revert the file to exactly how it was. 

E.g, The `PULL_REQUEST_TEMPLATE.md` doesn't have space on line 2. Now it is part of your PR cos it is seeing it that you added extra new line space.

For the `STYLEGUIDE.md` it is showing that your added new line spaces and the tab space aren't 2 it is 4. 

Kindly look that the changes and revert it to the way it was. Once you do it, it won't be part of your PR

**Comment by Israel-Force** [comment]
I think it's all set now

File: submissions/israelOladosu/php-slim/app/controllers/ArticleController.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+session_start();
+
+class ArticleController
+{
+    public function index(Request $request, Response $response)
+    {
+        $articles = Article::all();
+
+        $response->getBody()->write(json_encode(["status" => "success", "data" => [$articles]]));
```
Comment (adaugochi) [inline submissions/israelOladosu/php-slim/app/controllers/ArticleController.php:17]: Remove the square bracket. the `data` value should be an `object` not an `array`. The same goes for other places

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done

File: submissions/israelOladosu/php-slim/app/controllers/ArticleController.php (hunk 1 above)
Comment (Israel-Force) [reply submissions/israelOladosu/php-slim/app/controllers/ArticleController.php:17]: Done now. Thank you

**Review by adaugochi — APPROVED** [review]

**Pull request merged**